Sources/Sharing/Shared.swift

```diff
@@ -102,6 +102,7 @@ public struct Shared<Value> {
   ///   }
   /// }
   /// ```
+  #if compiler(>=6)
   public var wrappedValue: Value {
```

Comment:
It's not possible to surround just the `set { … }` in the `#if` check. At least not in Swift 5.10.